docs/usage.rst

```diff
@@ -219,6 +219,33 @@ Reporting on non-accounting data (queries)
 ------------------------------------------
 Check the module `mis_builder_demo <https://github.com/OCA/mis-builder/tree/10.0/mis_builder_demo>`_ to see how to create specific series of data for Committed purchases not yet invoiced.
 
+Example of query: we want a report showing the Total amount (HVAT) of all sales order confirmed during the current month and the previous month.
+
+Create a query:
+
+* Name: select a name (it must be a python expression, so no space in the name)
+* Model: look for the model on which you want to do the report. For example : Sale Order
+* Field to fetch: select in the list the field from the Model that will be used in the report. For example: Untaxed Amount
+* Fetched fields name: this will show the name of the field to use in the KPI expression later on. For example: amount_untaxed
+* Aggregate: choose between nothing, Sum, Max, Average or Min. Use either this aggregate field or you will need to add some aggregate expression in the KPI, because the query will give a list, not a number.
+* Date field: choose a date field available on the Model. For example: confirmation date of the sale order.
+* Domain: this is optional. Use a domain (as definded in classic Odoo). For example: [("partner_id.country_id.code","=","US")]
+
+.. image:: _static/images/query_1.png
+   :width: 1800
+
+In the KPI expression, you can now use the fields of the queries.
+For example:
+sum([s.amount_untaxed for s in sales_order_amount])
+
+.. image:: _static/images/query_2.png
+   :width: 1800
+
+In a reporting instance, the result is as follows:
+
+.. image:: _static/images/query_3.png
+   :width: 1800
+
```

Comment:
Another case is to use the internal way with: the name of the query and the fetched field name.

<img width="1122" alt="query_3" src="https://user-images.githubusercontent.com/6325713/65949791-28ca5f00-e43d-11e9-9e0d-d870ab1276ad.png">

<img width="1126" alt="query_4" src="https://user-images.githubusercontent.com/6325713/65949792-28ca5f00-e43d-11e9-885f-7a6a4f459417.png">